### PR TITLE
Fixes #956 content for help section

### LINKED ISSUE #956: Contents for help section in search settings
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Current behavior:**

The help section in settings -> search settings, re-directs to home page.
**Expected behavior:**

We need to design help page which will help users to troubleshoot frequent problems.
**Screenshots of the issue:**

Can I work on this? Any suggestions?

### PR DESCRIPTION
**Pull request by prabhupant**
Fixes #956 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [x] Added Surge preview link

#### Changes proposed in this pull request:
add content for help section

- https://pr-1186-fossasia-susper.surge.sh

-
new gif

### CODE REVIEW

**Comment by prabhupant** [comment]
@praveenojha33 please review again. travis ci build has passed now

**Comment by prabhupant** [comment]
@praveenojha33 sure. though i don't have that much experience with frontend so it might take some time :+1:

File: src/app/help/help.component.css [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+.navbar-right {
+  border-top:none;
+}
+.nav-terms {
+	padding: 0px;
+    clear: both;
+    margin: -57px 0 0 0
+}
+
+
```
Comment (simsausaurabh) [inline src/app/help/help.component.css]: Try to write clean code, remove extra unwanted spaces.

File: src/app/help/help.component.html [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+                <li><a routerLink="/about">About</a></li>
+                <li><a routerLink="/contact">Contact</a></li>
+            </ul>
+        </div>
+
+    </div>
+</nav>
+
+
+
```
Comment (simsausaurabh) [inline src/app/help/help.component.html]: Try to write clean code, remove extra unwanted spaces.

File: src/app/help/help.component.html [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+              <p>Susper is a decentral Search Engine that uses the peer to peer system 'YaCy' and 'Apache Solr' to crawl and index search results.</p>
+              <p>For more information on how Susper works, visit our <a routerLink="/about">About Us</a></p>
+
+          </li>
+
+        </ul>
+
+
+
+
```
Comment (simsausaurabh) [inline src/app/help/help.component.html]: Try to write clean code, remove extra unwanted spaces. Please follow it everywhere!

**Comment by prabhupant** [comment]
@praveenojha33 @simsausaurabh so i've worked on improving the help page to make it like google's. Please review this gif so that i can later work on improving it if you want and also squash the commits and push the changes. And also i have one question - for those links in the accordion, should i make a modal for each one of them?

help section

**Comment by prabhupant** [comment]
@praveenojha33 @simsausaurabh please review. about adding answers to the questions, ModalComponent has been depreciated. So I can use BsModalComponent, but we might have to make changes in the contact component too then. Need discussion

**Comment by prabhupant** [comment]
@praveenojha33 please review again. I tried adding the thumbnails but they were looking odd when compared to google's :sweat_smile: . Otherwise, I've made all the changes.

**Comment by prabhupant** [comment]
@praveenojha33 @simsausaurabh please merge :sweat_smile:

**Pull request merged**